README.md

```diff
@@ -176,17 +176,11 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
 
 **More Information on Data Sources from [Table 2](#table-2-canopy-app-required-input-variables):**
 
-**Global GFS meteorological files are available on AWS:**
-```
-https://noaa-oar-arl-nacc-pds.s3.amazonaws.com/inputs/
-```
+**Global GFS meteorological files and global canopy files for 2022 are available on AWS:**
```

Comment:
```suggestion
**Global GFS meteorological files and global canopy files for 2022 are available on [AWS](https://registry.opendata.aws/noaa-oar-arl-nacc-pds/):**
```

Reply:
Sorry for any confusion, Ill leave it up to you.  I think embedded links indeed should point to AWS registry and we show explict paths to AWS buckets /inputs and /geo-files